packages/featured-slider/index.js

```diff
@@ -133,100 +135,90 @@ class FeaturedSlider extends Component {
+                  <div className={s.featureContent}>
+                    {single && (
+                      <div className={s.singleLogo}>
+                        <Image url={feature.logo.url} alt={feature.logo.alt} />
                       </div>
-                      <div className="feature-content g-type-body">
-                        {single && (
-                          <div className="single-logo">
-                            <Image
-                              url={feature.logo.url}
-                              alt={feature.logo.alt}
-                            />
-                          </div>
-                        )}
-                        <h3
-                          className="g-type-display-4"
-                          dangerouslySetInnerHTML={{
-                            __html: feature.heading,
-                          }}
-                        />
-                        <p
-                          className="g-type-body"
-                          dangerouslySetInnerHTML={{
-                            __html: feature.content,
-                          }}
-                        />
-                        <Button
-                          className="g-btn"
-                          theme={{
-                            variant: 'secondary',
-                            background: theme,
-                          }}
-                          linkType={feature.link.type}
-                          title={feature.link.text}
-                          url={feature.link.url}
-                        />
-                      </div>
-                    </div>
+                    )}
+                    <h3
+                      className={classNames(s.contentHeading, {
+                        [s.dark]: theme === 'dark',
+                      })}
+                      dangerouslySetInnerHTML={{
+                        __html: feature.heading,
+                      }}
+                    />
+                    <p
+                      className={classNames(s.contentBody, {
+                        [s.dark]: theme === 'dark',
+                      })}
+                      dangerouslySetInnerHTML={{
+                        __html: feature.content,
+                      }}
+                    />
+                    <Button
+                      className={s.button}
+                      theme={{
+                        variant: 'secondary',
+                        background: theme,
+                      }}
+                      linkType={feature.link.type}
+                      title={feature.link.text}
+                      url={feature.link.url}
+                    />
                   </div>
-                ))}
+                </div>
               </div>
-            </div>
+            ))}
           </div>
         </div>
-      </section>
+      </div>
     )
   }
 }
 
+function FeaturedSlider({ theme, heading, className, ...restProps }) {
```

Comment:
I've split out the "padded section" wrapper of this component. This aligns both with resolving duplicative implementations between this component and `case-study-slider`, and with the task to [remove external padding and margin from components](https://app.asana.com/0/1100423001970639/1196250350088041/f).